website/src/pages/docs/basics/client.mdx

```diff
@@ -233,6 +235,14 @@ From now on out, every time we need a new field in the `Avatar` component we can
 and trust that the query is updated automatically and our data is passed into the component by menans of
 `<Avatar user={result.data.user} />`.
 
+### `useFragment` and `FragmentType`
+
+You'll see that we use this `useFragment` function in our components that have a fragment describing their
+data-requirements, this helper ensures that we remove any extraneous fields and type the output correctly.
+Another thing worth noting is how we type the `props`, we use this `FragmentType` helper which ensures that
+the parent-component uses the fragment and passes that into props, this means that if the parent-component
+would define all the fields of the fragment manually and not spread it that we would show a warning.
```

Comment:
Add a callout below here that explains you can/have to use `useFragment` also in server components and that it's not _really_ a hook. (I have a preference to change this API eventually 😬)

Reply:
I added it!